src/Io/MultipartParser.php

```diff
@@ -68,7 +68,7 @@
     private $emptyCount = 0;
 
     /**
-     * @param int|null $uploadMaxFilesize
+     * @param int|string|null $uploadMaxFilesize
```

Comment:
LGTM, same change is required for `RequestBodyParserMiddleware` :+1:

Reply:
Done